router/core/websocket.go

```diff
@@ -792,6 +789,9 @@ func (h *WebSocketConnectionHandler) parseAndPlan(payload []byte) (*ParsedOperat
 		opContext.parsingTime = time.Since(startParsing)
 	}
 
+	opContext.name = operationKit.parsedOperation.Request.OperationName
```

Comment:
I'd use strings.clone or similar here to make a dedicated copy of the name. Operationkit is re-used.

Reply:
We already ensure [here](https://github.com/wundergraph/cosmo/blob/f6a4224a2370e8eb6e36598a22f60a3eee83f055/router/core/operation_processor.go#L489) that we copy the value.